2-mandatory/3-shopping-list.js

```diff
@@ -20,6 +20,16 @@ let pantry = {
 
 function createShoppingList(recipe) {
   // write code here
+  const newShoppingList = {};
+  newShoppingList.name = recipe.name;
+  newShoppingList.items = [];
+  const pantryList = Object.values(pantry).flat();
```

Comment:
greate job, you can also use concat for merging two arrays 
`pantry.fridgeContents.concat(pantry.cupboardContents)`

Reply:
Hi Ellie, thank you for your code review.